Changelog.md

```diff
@@ -2,16 +2,17 @@
 
 ## Next
 
-* Add `burn : <system>Nat -> Nat` to `Cycles` (#228)
-* Add `isEmpty : _ -> Bool` to `Blob` and `Text` (#263)
-* Fix bug in `Iter.step()`
-* Fix bug in `Array.enumerate()` and `VarArray.enumerate()`
-* Fix argument order for `Result.compare()` and `Result.equal()`
-* Adjust `Random` representation to allow persistence in stable memory
+* Add `isReplicated : () -> Bool` to `InternetComputer` (#213).
+* Add `burn : <system>Nat -> Nat` to `Cycles` (#228).
+* Add `isEmpty : _ -> Bool` to `Blob` and `Text` (#263).
+* Fix bug in `Iter.step()` (#???).
```

Comment:
Can we add the PR numbers here?

Reply:
I want to update the Changelog in a separate PR